### ISSUE TITLE
feat: Improve Documentation

### ISSUE THREAD

**Issue by adiati98** [shesharpnl/knowledge-hub#7]
## Description

I've gone through the repository and I see that there are rooms for documentation improvement. The aim is to give clear explanations and instructions for contributors, especially those who are new to open source.
Right now, the README is a bit challenging to navigate for people who come to this repository for the first time. And there are also essential things missing in this repository.

### Add a license

Without a license, this repository is automatically licensed under "All Rights Reserved" which means that no one may use, modify, or redistribute anything in the project unless you as the owner gives them permission to.

So you need to add a `LICENSE` file to this repo and link the license in its section on the README. [This website](https://www.tldrlegal.com/) is very helpful in explaining Software Licenses in Plain English. [MIT License](https://www.tldrlegal.com/license/mit-license) is one of licenses that's used most in open source.

### Add a code of conduct

Having a code of conduct helps to maintain a safe and welcoming space for contributors. It will also help you to moderate contributions and communication going on.

You can use your community's code of conduct and put it in a `CODE_OF_CONDUCT.md` file. Or you can use [a template from GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project#adding-a-code-of-conduct-using-a-template) to add one.

### Change the title "Suggesting a Resource" to "Contributing Guide"

I see this section contains:
- Inviting contributors to submit resources.
- Navigate contributors to "How to submit your suggestion" — which contains the instructions to contribute.

So having a straightforward title will give clarification about the section.

### Replace the `knowledge-hub/how-to/submit-your-suggestion.md` with `CONTRIBUTING.md` that will live in the root.

`CONTRIBUTING.md` is a well-known file accross open-source projects that serves as a guide for contributors. Replacing the `knowledge-hub/how-to/submit-your-suggestion.md` with `CONTRIBUTING.md` will help SheSharp members to get familiar with the file and know where to find it in a repository when they contribute to other OSS projects.

You want to also mention here what type of resources you are accepting for contributions.

### Move the "[Randstad Housing Resources](https://github.com/shesharpnl/knowledge-hub#randstad-housing-resources)" and "[Organizations that help women and non-binary people break into tech](https://github.com/shesharpnl/knowledge-hub#organizations-that-help-women-and-non-binary-people-break-into-tech)" in the README to their own files.

There is the table of content in the README. However, it's confusing to see various things going on in the README.

For example, in the introduction I read this:

> The resources collected here cover different areas, such as software development, data analysis, product management and quality assurance.

But we do have other resources outside what stated there.

This suggestion also comes from a consideration that potentially the list in the resources will get longer and SheSharp may add new resources type, for example, "Job hunt resources", "Resources for moms in tech", etc.

My suggestion is to move the resources to their own files and we can link these files on the README, but not having their content in the README.

### [Social Media](https://github.com/shesharpnl/knowledge-hub#social-media) section

- Adjust the title to "SheSharp Members' Social Media" or something like that to clarify that only the SheSharp members are on the list, except if this repository is accepting contributions from outside of the community.
- Having the social media section (Instagram, LinkedIn, Twitter, GitHub profiles, etc.) on the README is great. But when it's getting much longer, you want to consider to move it to its own file.

### How to communicate

I see this sentence in the contributing guide:

> If you need any help, feel free to raise an issue.

It would be best to give this sentence more attention so contributors can find it quickly.

My suggestion to have its own section on the README called "Do You Have Questions?" or "Community" or anything else that suit your need.

This section will contain:
- When contributors need help, have questions or ideas, where do they need to go? — Slack (and which channel), raise an issue, or using [Discussion board](https://github.com/features/discussions)?
- Optional — if someone wants to contribute but not a SheSharp member, how to join SheSharp?

### Is this repository receiving contributions from members only?

If this repo receives contributions from members only, you want to make it clear by mentioning about it in the introduction on the README. Otherwise, you can state that you encourage everyone to contribute.

### Clarify the [Step by step](https://github.com/shesharpnl/knowledge-hub/blob/main/how-to/submit-your-suggestion.md#step-by-step) section.

We don't want to assume that everybody knows the syntax on how to clone, create a new branch, commit, etc. So a clarification on how to do them would help beginners in open source to contribute to this repo.

We can do so by:

**Linking each step to a resource**. For example:

3. [Create a branch](https://www.shellhacks.com/git-create-new-branch-and-checkout/) with a name that relates with your contribution.

**OR**

**Manually write the instructions**. For example:

3. Create a branch with a name that relates with your contribution.
      Run this command in your terminal to create a new branch and navigate to it:
  
      ```bash
      git checkout -b <branch-name>
      ```

      For example:

      ```bash
      git checkout -b feat/add-ayu-linkedin
      ```

**Comment by adiati98** [comment]
Hi @mariahlaqua and maintainers team,
I'm down to help you break down this issue into small ones and work on them if you give them a go 🙂.

Just let me know which ones you approve. 

Thank you 😊.

**Comment by mariahlaqua** [comment]
Great issue @adiati98 , thank you so much. Looking forward to seeing some PRs to improve this repo.

**Comment by saragerion** [comment]
Thanks for creating this issue. I agree with everything! Great suggestions.

**Comment by adiati98** [comment]
Hi @mariahlaqua and @saragerion, here is the list of our tasks so far 😊

**Tasks done**:
- [x] Add a code of conduct
- [x] Change the title "Suggesting a Resource" to "Contributing Guide"
- [x] Replace the `submit-your-suggestion.md` with `CONTRIBUTING.md` that will live in the root.
- [x] Move the "Randstad Housing Resources" and "Organizations that help women and non-binary people break into tech" in the README to their own files.
- [x] How to communicate
- [x] Clarify the [Step by step](https://github.com/shesharpnl/knowledge-hub/blob/main/how-to/submit-your-suggestion.md#step-by-step) section.

**Tasks on progress**:
- [ ] Add a license --> License is not supported in `.github` repository. So we need to create license for each repo and mention it in a section at the README.
- [ ] [Social Media](https://github.com/shesharpnl/knowledge-hub#social-media) section.
- [ ] Is this repository receiving contributions from members only?

**Comment by mariahlaqua** [comment]
@adiati98 - I think anyone can contribute for now
<img width="328" alt="Screenshot 2023-09-28 at 22 41 55" src="https://github.com/shesharpnl/knowledge-hub/assets/78166655/f5ed712b-1cf7-4d78-a80b-029d948387f8">
It looks like we can't add a license to the knowledge hub - or at least I cannot.

**Comment by mariahlaqua** [comment]
Maybe @saragerion has the permissions for adding a license to this one?

**Comment by saragerion** [comment]
>  Maybe @saragerion has the permissions for adding a license to this one?

Thanks for calling it out. Our licenses were either not present or inconsistent across repositories. I added an MIT license in all our repos.

**Comment by saragerion** [comment]
> Is this repository receiving contributions from members only?

Nope, everyone is welcome to make contributions, also people who are not community members

**Comment by adiati98** [comment]
> > Maybe @saragerion has the permissions for adding a license to this one?
> 
> Thanks for calling it out. Our licenses were either not present or inconsistent across repositories. I added an MIT license in all our repos.

Hey @saragerion,

We don't have to have same license accross the repos. Type of license depends on the repo's need.
I also saw some repos with multiple licenses if necessary such as [GitHub docs here](https://github.com/github/docs#license).

**Comment by CBID2** [comment]
I can do the social media section if that task is free for the taking @saragerion and @adiati98

**Comment by adiati98** [comment]
> I can do the social media section if that task is free for the taking @saragerion and @adiati98

@saragerion has confirmed that everyone can contribute to this repo. If it also means that the social media section also open for public, then our tasks are done, @CBID2.

Social media section is part of @mariahlaqua's tasks.

So, let's hear the confirmation on this :)

**Comment by mariahlaqua** [comment]
@adiati98 @saragerion I wanted to close the other PR before I made the Social Media changes. I'm doing those now :)

**Comment by CBID2** [comment]
Ok thanks @adiati98 and @mariahlaqua! :)
